src/MySqlConnector/MySqlConnector.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>A fully async MySQL ADO.NET provider for .NET and .NET Core</Description>
+    <Copyright>Copyright 2016 Bradley Grainger</Copyright>
+    <AssemblyTitle>Async MySQL Connector</AssemblyTitle>
+    <VersionPrefix>0.13.0</VersionPrefix>
+    <Authors>Bradley Grainger;Caleb Lloyd</Authors>
+    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AssemblyName>MySqlConnector</AssemblyName>
+    <PackageId>MySqlConnector</PackageId>
+    <PackageTags>mysql;async;ado.net;database;netcore</PackageTags>
+    <PackageReleaseNotes>Add MySqlBulkLoader. GitHub issue #15</PackageReleaseNotes>
+    <PackageProjectUrl>https://github.com/mysql-net/MySqlConnector</PackageProjectUrl>
+    <PackageLicenseUrl>https://opensource.org/licenses/MIT</PackageLicenseUrl>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/mysql-net/MySqlConnector.git</RepositoryUrl>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
+    <PackageReference Include="System.Buffers" Version="4.0.0" />
+    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.0.0" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.0.0" />
+    <Reference Include="System" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Transactions" />
+    <Reference Include="Microsoft.CSharp" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
```

Comment:
You can add : <PackageReference Remove="NETStandard.Library" /> 

to remove the implicit reference to .NETStandard 1.6.1 made by the SDK.